test/pylib/cpp/common_cpp_conftest.py

```diff
@@ -71,13 +72,19 @@ def get_disabled_tests(config: dict, modes: list[str]) -> dict[str, set[str]]:
 
 def read_suite_config(directory: Path) -> dict[str, str]:
     """
-    Helper method that will return the configuration from the suite.yaml file
+    Helper method that returns the configuration from the test_config.yaml file.
+    It can be that there's no test_config.yaml that means there are no additional parameters for Scylla itself or no
+    filtering on what modes tests should be executed or skipped
     """
-    with open(directory / 'suite.yaml', 'r') as cfg_file:
-        cfg = yaml.safe_load(cfg_file.read())
-        if not isinstance(cfg, dict):
-            raise RuntimeError('Failed to load tests: suite.yaml is empty')
-        return cfg
+    config = directory / TEST_CONFIG
+    if config.exists() and config.stat().st_size:
+        with config.read_text(encoding='utf-8') as cfg_file:
```

Comment:
Sorry, but it doesn't work like this: `.read_text()` returns `str`.

CI passed and that means this code not executed during the test run.